mono/metadata/assembly.c

```diff
@@ -557,6 +557,21 @@ mono_assembly_getrootdir (void)
 }
 
 /**
+ * mono_native_getrootdir:
+ * 
+ * Obtains the root directory used for looking up native libs (.so, .dylib).
+ *
+ * Returns: a string with the directory, this string should be freed by
+ * the caller.
+ */
+G_CONST_RETURN gchar *
```

Comment:
shouldn't be `const` as the caller is required to free it (that was silently fixed in 1fb083d2d4fa63b41b1c97bb61cd5782a62c910b)